docs/specviz/displaying.rst

```diff
@@ -90,16 +94,6 @@ Interactive Pan/Zoom (Desktop or Notebook Interface)
 You can find the following Pan/Zoom tools available in the viewer toolbar
 on the top right of the viewer.
 
-
-Home
```

Comment:
This was completely removed? Was it on purpose?

Reply:
It looks like it was a duplicate section, you can see above there's a home heading with the same text.